vocabularies/voc4cat/0007219.ttl

```diff
@@ -5,16 +5,18 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://w3id.org/nfdi4cat/voc4cat_0007219>
     a skos:Concept ;
     dcterms:identifier "voc4cat_0007219"^^xsd:token ;
-    skos:broader
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0007219> ;
+    skos:definition "The electrical potential difference between two points in a circuit is the cause of the flow of a current."@en ;
```

Comment:
This definition seems incorrect. A potential is not the same as a potential difference. Is the key point that a potential refers to a reference?

Reply:
I am not an expert, but i think these things are used interchangably

Reply:
I am also no expert in this, @nmoust can you help?

Reply:
I am not an expert either but maybe we can can one concept labeled "potential difference" with a definition: the difference between two potentials. And then we have separate definitions for e.g., "electrical potential" and "chemical potential" which includes difference of the respective potential between a reference point and a different point.

Reply:
A condensed version of several sources: 
"Defined as the amount of work needed per unit of electric charge to move the charge from a reference point to a specific point in an electric field. By definition, the electric potential at the reference point is zero units. A typical reference point is earth (ground)."